pset2/caesar/caesar.c

```diff
@@ -12,7 +23,7 @@ int main(int argc, string args[])
         return 1;
     }
 
-    int k = atoi(args[1]) % 26; // if k is > 26, store the division remainder instead
+    int k = atoi(args[1]) % LETTERS_COUNT; // if k is > 26, store the division remainder instead
```

Comment:
//Why is the modula result stored in k  when you are going to use this "int ci = (pi + k) % LETTERS_COUNT" later ?.